sakai-evaluation-tool/src/webapp/component-templates/render_text_item.html

```diff
@@ -35,8 +35,10 @@ <h5 class="title">
 				<span rsf:id="itemText" class="label">Item text label.</span>
 
 				<!-- Previous
```

Comment:
Thanks, I tested that all of this compiles and passes the tests but I didn't actually start it up yet to see if these actually all work. For some things like this I'm not sure the history of the changes yet or why it would be commented out now.